### PR TITLE
[FIX] stock: cache is not clear after merge quants

### PR DESCRIPTION
**Pull request by fmdl**
**Description of the issue/feature this PR addresses:**
Merge quant can delete some quant by direct SQL (without using unlink()).
The cache is not cleared.

@nim-odoo
@amoyaux 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by amoyaux** [comment]
robodoo r+

**Comment by amoyaux** [comment]
robodoo retry

**Comment by amoyaux** [comment]
robodoo retry

File: addons/stock/models/stock_quant.py [hunk 1]
```diff
@@ -531,6 +531,7 @@ def _merge_quants(self):
         try:
             with self.env.cr.savepoint():
                 self.env.cr.execute(query)
+                self.invalidate_cache()
```
Comment (ryv-odoo) [inline addons/stock/models/stock_quant.py:534]: The cache is already invalidate by the savepoint no ?

Comment (ryv-odoo) [reply addons/stock/models/stock_quant.py:534]: My bad, you alright.

**Pull request closed without merging**